quest_test/test_workflow_metrics.py

```diff
@@ -0,0 +1,42 @@
+import asyncio
+import pytest
+
+from quest.manager import WorkflowManager
+from quest.persistence import InMemoryBlobStorage
+from quest.serializer import NoopSerializer
```

Comment:
I'd like to move the logic for creating a test-appropriate workflow manager to the test utils module.